tests/util.spec.coffee

```diff
@@ -117,3 +121,35 @@ describe 'Pine option merging', ->
 		m.chai.expect(
 			-> mergePineOptions(unknownKey: 'value', {})
 		).not.to.throw()
+
+describe 'ImgMakerHelper', ->
+
+	ROOT_URL = 'https://img.resin.io'
+
+	beforeEach =>
+		@requestStub = send: m.sinon.stub().returns(new Promise(
+			-> # never return, don't worry about responses
```

Comment:
This is _very_ annoying, I've hit it before too, and it doesn't seem to come up in local linting... I'll going to find a proper way to fix it now.